variables.tf

```diff
@@ -52,3 +52,8 @@ variable "use_AmazonEC2ContainerServiceforEC2Role_policy" {
   default     = true
 }
 
+variable "security_group_ids" {
+  description = "A list of aws security group ids to launch resources in"
```

Comment:
Maybe nit-picky. You don't launch resources in a security group. A security group defines what a resource can reach on the network. Your wording here makes it sound like it's meant to be a target group in an ALB definition.

Reply:
Nah, this helps my understanding. I'll change the wording.